src/Utils.ml

```diff
@@ -20,6 +20,9 @@ module List = struct
                      ~(f : 'a -> 'b -> string) : string =
     String.concat ~sep (List.map2_exn l1 l2 ~f)
 
+  let sort ~(cmp : 'a -> 'a -> int) (l : 'a list) : 'a list =
+    List.sort ~compare:cmp l
```

Comment:
I'd remove this. Why not use `List.sort` directly?

Reply:
Because there was a keyword name change from ~cmp to ~compare? I might be mistaken.